app/Resources/views/card/new.html.twig

```diff
@@ -4,93 +4,140 @@
+                      {% for i in 0..23 %}
+                        <option value="{{i}}">
+                          {% if i <= 9 %}0{{i}}{% else %}{{i}}{% endif %}
+                        </option>
+                      {% endfor %}
+                      </select>
+                        :
+                      <select id="appbundle_card_datetime_time_minute" name="appbundle_card[datetime][time][minute]">
+                      {% for i in 0..59 %}
+                        <option value="{{i}}">
+                          {% if i <= 9 %}0{{i}}{% else %}{{i}}{% endif %}
+                        </option>
+                      {% endfor %}
+                      </select>
+                  </div>
+              </div>
+          </div>
+
+          <div class="form-group col-md-4">
+            <label for="appbundle_card" class="required">Title</label><br>
+            <input type="text" id="appbundle_card" class="form-control" name="appbundle_card[title]" required="required">
+          </div>
+
+          <div class="form-group col-md-5">
+            <label for="appbundle_card" class="required">Description</label><br>
+            <textarea id="appbundle_card" class="form-control" name="appbundle_card[description]" required="required"></textarea>
+          </div>
+
+      </div>
+
+      <div class="form-row">
+        <div class="form-group col-md-12">
+          Project :<br>
+          {% for project in projects %}
+            <div class="form-check form-check-inline">
+              <label class="form-check-label required">
+                <input type="radio" class="form-check-input" value="{{project.name}}" name="appbundle_card[project]"
+                {% if project.name == card.project %}checked="checked"{% endif %}> {{project.name}}
+              </label>
             </div>
-            <div>
-                <label class="required">Type</label>
-                {% set counter = 0 %}
-                {% for type in types %}
-                    {% set counter = counter + 1 %}
-                    <input type="radio" id="appbundle_card_type_{{counter}}" value="{{type.name}}" name="appbundle_card[type]"
-                    {% if type.name == 'task'%}checked="checked"{% endif %}> {{type.name}}
-                {% endfor %}
+          {% endfor %}
+        </div>
+      </div>
+
+      <div class="form-row">
+        <div class="form-group col-md-12">
+          Type :<br>
+          {% set counter = 0 %}
+          {% for type in types %}
+            <div class="form-check form-check-inline">
+              <label class="form-check-label required">
+                {% set counter = counter + 1 %}
+                <input type="radio" class="form-check-input" id="appbundle_card_type_{{counter}}" value="{{type.name}}" name="appbundle_card[type]"
+                {% if type.name == 'task'%}checked="checked"{% endif %}> {{type.name}}
```

Comment:
I've also fixed this auto selection of project and card type value here https://github.com/sensorario/kanbanoise/pull/29

Reply:
Pull https://github.com/sensorario/kanbanoise/tree/Fukotaku-master. I've fixed conflicts

Reply:
Oups, that attribut value it's for edit, not new. Sorry ^^'